src/dbus_api.rs

```diff
@@ -120,7 +120,29 @@ fn list_pools(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
 }
 
 
+
 fn create_filesystems(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
+    let message: &Message = m.msg;
+    let mut iter = message.iter_init();
+
+    if iter.arg_type() == 0 {
+        return Err(MethodErr::no_arg());
+    }
+
+    let _volumes: Array<(&str, &str, &str), _> = try!(iter.read::<Array<(&str, &str, &str), _>>()
+        .map_err(|_| MethodErr::invalid_arg(&0)));
+
```

Comment:
yes.

all the println! calls are going to be removed once I get it working.  this is intended as
a place holder for making the calls to engine.